packages/engine/src/avatar/components/LoopAnimationComponent.ts

```diff
@@ -187,7 +215,19 @@ export const LoopAnimationComponent = defineComponent({
       const stop = () => {
         loopAnimationComponent.paused.set(true)
         loopAnimationComponent.time.set(0)
-        loopAnimationComponent._action.value?.stop()
+
+        // Stop the current action
+        if (loopAnimationComponent._action.value) {
+          loopAnimationComponent._action.value.stop()
+        }
+
+        // Also stop all actions in the mixer to be thorough
+        if (animComponent?.mixer?.value) {
+          animComponent.mixer.value.stopAllAction()
+        }
+
+        // Set activeClipIndex to -1 (None) to indicate no clip is selected
+        loopAnimationComponent.activeClipIndex.set(-1)
```

Comment:
I don't think we want to change the activeClipIndex, that is what specifies which animation should run and should only be set via the editor UI